targets/targets.json

```diff
@@ -1139,6 +1139,10 @@
                 "help": "Define if a Low Speed External xtal (LSE) is available on the board (0 = No, 1 = Yes). If Yes, the LSE will be used to clock the RTC, LPUART, ... otherwise the Low Speed Internal clock (LSI) will be used",
                 "value": "1"
             },
+            "rtc_clock_source": {
+                "help": "Define the RTC clock source. USE_RTC_CLK_LSE_OR_LSI, USE_RTC_CLK_LSI, USE_RTC_CLK_HSE. LPTICKER is not available for HSE and should be removed from the target configuration.",
```

Comment:
USE_RTC_CLK_LSI not used ?

Reply:
How do you mean not used?

Reply:
- USE_RTC_CLK_LSE_OR_LSI: OK, this is the default value, i.e. code before this PR
- USE_RTC_CLK_HS: this is the goal of this PR
- USE_RTC_CLK_LSI: no specific code for this choice ?

Reply:
There **is** code for it, just search for it in the changed files. It is to enforce using LSI, even though LSE is available. I don't know if it is ever a use case, but giving the option does not hurt. I could remove it if you like.

Reply:
OK, I let you pushing the minor updates we have seen ?
Thx